Bottom: 4120b0ceb59ecdb17c7a38e26d150aac56b8929d
Top:    bd983e0fdea1bc8a92a045adc118a4b5f970b337
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-22 11:30:36 -0500

Normalize variable names

Improve consistency in variable names:
- ensure variables within roles always begin with the role name
- no global variables containers holding variables for multiple roles



---
diff --git a/ansible-playbook/group_vars/all/credentials.yml b/ansible-playbook/group_vars/all/credentials.yml
index 6e11e08..41773f2 100644
--- a/ansible-playbook/group_vars/all/credentials.yml
+++ b/ansible-playbook/group_vars/all/credentials.yml
@@ -1,7 +1,7 @@
 ---
 # passwords
-ldap_admin_password:                 '{{ lookup("password", playbook_dir + "/credentials/ldap/cn=admin,dc=novatech length=20") }}'
-ldap_proxyagent_password:            '{{ lookup("password", playbook_dir + "/credentials/ldap/cn=proxyagent,dc=novatech length=20") }}'
+openldap_admin_password:                 '{{ lookup("password", playbook_dir + "/credentials/ldap/cn=admin,dc=novatech length=20") }}'
+openldap_proxyagent_password:            '{{ lookup("password", playbook_dir + "/credentials/ldap/cn=proxyagent,dc=novatech length=20") }}'
 wiki_db_root_password:               '{{ lookup("password", playbook_dir + "/credentials/wiki/db/root length=20") }}'
 wiki_db_password:                    '{{ lookup("password", playbook_dir + "/credentials/wiki/db/user length=20") }}'
 wiki_email_password:                 '{{ lookup("password", playbook_dir + "/credentials/wiki/email/wiki@novatech-llc.com length=20") }}'
diff --git a/roles/buildbot-master/defaults/main.yml b/roles/buildbot-master/defaults/main.yml
index 64802d0..809062a 100644
--- a/roles/buildbot-master/defaults/main.yml
+++ b/roles/buildbot-master/defaults/main.yml
@@ -32,7 +32,7 @@ buildbot_dv_name: '{{ docker_name_prefix }}buildbot_DV'
 
 # restore directories to temporarly store data being restored into docker containers
 buildbot_docker_backup_dir: '{{ docker_backup_dir }}/buildbot'
-host_buildbot_docker_restore_dir: '{{ bacula_dest }}{{ buildbot_docker_backup_dir }}'
+buildbot_docker_restore_dir: '{{ bacula_dest }}{{ buildbot_docker_backup_dir }}'
 
 buildbot_build_args:
   BUILDBOT_UID: '1000'
diff --git a/roles/docker-exim4/defaults/main.yml b/roles/docker-exim4/defaults/main.yml
index 7262e16..e760fb8 100644
--- a/roles/docker-exim4/defaults/main.yml
+++ b/roles/docker-exim4/defaults/main.yml
@@ -31,7 +31,7 @@ exim4_dv_name: '{{ docker_name_prefix }}exim4_DV'
 
 # restore directories to temporarly store data being restored into docker containers
 exim4_docker_backup_dir: '{{ docker_backup_dir }}/EXIM4'
-host_exim4_docker_restore_dir: '{{ bacula_dest }}{{ exim4_docker_backup_dir }}'
+exim4_docker_restore_dir: '{{ bacula_dest }}{{ exim4_docker_backup_dir }}'
 
 # files restored from tape
 exim4_backup_file: '/exim4.tar'
diff --git a/roles/docker-exim4/tasks/restore.yml b/roles/docker-exim4/tasks/restore.yml
index e863805..6910243 100644
--- a/roles/docker-exim4/tasks/restore.yml
+++ b/roles/docker-exim4/tasks/restore.yml
@@ -6,7 +6,7 @@
 
 - name: exists - exim4 archive
   stat:
-    path: '{{ host_exim4_docker_restore_dir }}{{ exim4_backup_file }}'
+    path: '{{ exim4_docker_restore_dir }}{{ exim4_backup_file }}'
     get_checksum: False
     get_md5: False
   register: st_exim4_backup_file
@@ -22,7 +22,7 @@
 
 # - name: Set permissions for data-volume data
 #   file:
-#     path: '{{ host_exim4_docker_restore_dir }}{{ exim4_backup_file }}'
+#     path: '{{ exim4_docker_restore_dir }}{{ exim4_backup_file }}'
 #     state: touch
 #     owner: root
 #     group: root
@@ -30,7 +30,7 @@
 
 - name: exists - exim4 archive
   stat:
-    path: '{{ host_exim4_docker_restore_dir }}{{ exim4_backup_file }}'
+    path: '{{ exim4_docker_restore_dir }}{{ exim4_backup_file }}'
     get_checksum: False
     get_md5: False
   register: st_exim4_backup_file
@@ -39,7 +39,7 @@
 # restore the archive
 
 - name: restore archive
-  shell: 'cat {{ host_exim4_docker_restore_dir }}{{ exim4_backup_file }} | docker run -i --rm --volumes-from {{ exim4_dv_name }} {{ exim4_image_name }}:{{ docker_image_tag }} restore'
+  shell: 'cat {{ exim4_docker_restore_dir }}{{ exim4_backup_file }} | docker run -i --rm --volume {{ exim4_cv_name }}:/etc/exim4:z {{ exim4_image_name }}:{{ docker_image_tag }} restore'
   when: st_exim4_backup_file.stat.exists == True
 
 # *****************************************************************************
@@ -47,7 +47,7 @@
 
 - name: remove - exim4 archive
   file:
-    path: '{{ host_exim4_docker_restore_dir }}'
+    path: '{{ exim4_docker_restore_dir }}'
     state: absent
 
 - name: State file
diff --git a/roles/docker-gitlab/defaults/main.yml b/roles/docker-gitlab/defaults/main.yml
index b742c18..1a8478a 100644
--- a/roles/docker-gitlab/defaults/main.yml
+++ b/roles/docker-gitlab/defaults/main.yml
@@ -44,7 +44,7 @@ gitlab_db_dv_name: '{{ docker_name_prefix }}gitlab_db_DV'
 
 # restore directories to temporarly store data being restored into docker containers
 gitlab_docker_backup_dir: '{{ docker_backup_dir }}/GIT'
-host_gitlab_docker_restore_dir: '{{ bacula_dest }}{{ gitlab_docker_backup_dir }}'
+gitlab_docker_restore_dir: '{{ bacula_dest }}{{ gitlab_docker_backup_dir }}'
 
 # Network names
 docker_network_frontend: 'frontend'
diff --git a/roles/docker-gitlab/tasks/main.yml b/roles/docker-gitlab/tasks/main.yml
index ac5ffcc..3a43141 100644
--- a/roles/docker-gitlab/tasks/main.yml
+++ b/roles/docker-gitlab/tasks/main.yml
@@ -11,7 +11,7 @@
 - name: restore dir
   file:
     state: directory
-    path: '{{ host_gitlab_docker_restore_dir }}'
+    path: '{{ gitlab_docker_restore_dir }}'
     owner: root
     group: tape
     mode: 'u=rwx,g=rwx,o=rx'
diff --git a/roles/docker-gitlab/tasks/restore.yml b/roles/docker-gitlab/tasks/restore.yml
index 90c7934..c95eb56 100644
--- a/roles/docker-gitlab/tasks/restore.yml
+++ b/roles/docker-gitlab/tasks/restore.yml
@@ -6,7 +6,7 @@
 
 - name: find files
   files_in_dir:
-    path: '{{ host_gitlab_docker_restore_dir }}'
+    path: '{{ gitlab_docker_restore_dir }}'
   register: gitlab_backup
 
 - name: data from tape
@@ -20,12 +20,12 @@
 
 - name: find files
   files_in_dir:
-    path: '{{ host_gitlab_docker_restore_dir }}'
+    path: '{{ gitlab_docker_restore_dir }}'
   register: gitlab_backup
 
 # - name: File permissions
 #   file:
-#     path: '{{ host_gitlab_docker_restore_dir }}/{{ item }}'
+#     path: '{{ gitlab_docker_restore_dir }}/{{ item }}'
 #     state: touch
 #     owner: root
 #     group: root
@@ -46,7 +46,7 @@
 
 - name: Remove files
   file:
-    path: '{{ host_gitlab_docker_restore_dir }}'
+    path: '{{ gitlab_docker_restore_dir }}'
     state: absent
 
 - name: State file
diff --git a/roles/docker-mantisbt/defaults/main.yml b/roles/docker-mantisbt/defaults/main.yml
index e4df7a7..f2d3efa 100644
--- a/roles/docker-mantisbt/defaults/main.yml
+++ b/roles/docker-mantisbt/defaults/main.yml
@@ -41,7 +41,7 @@ mantisbt_db_dv_name: '{{ docker_name_prefix }}mantisbt_db_DV'
 
 # restore directories to temporarly store data being restored into docker containers
 mantisbt_docker_backup_dir: '{{ docker_backup_dir }}/MANTISBT'
-host_mantisbt_docker_restore_dir: '{{ bacula_dest }}{{ mantisbt_docker_backup_dir }}'
+mantisbt_docker_restore_dir: '{{ bacula_dest }}{{ mantisbt_docker_backup_dir }}'
 
 # files restored from tape
 mantisbt_static_backup_file: '/mantisbt.tar'
diff --git a/roles/docker-mantisbt/tasks/main.yml b/roles/docker-mantisbt/tasks/main.yml
index 0c2f902..e25c4a7 100644
--- a/roles/docker-mantisbt/tasks/main.yml
+++ b/roles/docker-mantisbt/tasks/main.yml
@@ -12,7 +12,7 @@
 - name: restore dir
   file:
     state: directory
-    path: '{{ host_mantisbt_docker_restore_dir }}'
+    path: '{{ mantisbt_docker_restore_dir }}'
     owner: root
     group: tape
     mode: 'u=rwx,g=rwx,o=rx'
@@ -176,7 +176,7 @@
 
 - name: cleanup - import mysql config
   file:
-    path: '{{ host_mantisbt_docker_restore_dir }}/{{ item }}'
+    path: '{{ mantisbt_docker_restore_dir }}/{{ item }}'
     state: absent
   with_items:
   - largefile.cnf
diff --git a/roles/docker-mantisbt/tasks/restore.yml b/roles/docker-mantisbt/tasks/restore.yml
index 323c254..78f66b9 100644
--- a/roles/docker-mantisbt/tasks/restore.yml
+++ b/roles/docker-mantisbt/tasks/restore.yml
@@ -6,21 +6,21 @@
 
 - name: exists - Static
   stat:
-    path: '{{ host_mantisbt_docker_restore_dir }}{{ mantisbt_static_backup_file }}'
+    path: '{{ mantisbt_docker_restore_dir }}{{ mantisbt_static_backup_file }}'
     get_checksum: False
     get_md5: False
   register: st_mantisbt_static_archive
 
 - name: exists - SQL (files table)
   stat:
-    path: '{{ host_mantisbt_docker_restore_dir }}{{ mantisbt_database_files_table_backup_file }}'
+    path: '{{ mantisbt_docker_restore_dir }}{{ mantisbt_database_files_table_backup_file }}'
     get_checksum: False
     get_md5: False
   register: st_mantisbt_files_table_SQL_file
 
 - name: exists - SQL (remaining tables)
   stat:
-    path: '{{ host_mantisbt_docker_restore_dir }}{{ mantisbt_database_backup_file }}'
+    path: '{{ mantisbt_docker_restore_dir }}{{ mantisbt_database_backup_file }}'
     get_checksum: False
     get_md5: False
   register: st_mantisbt_SQL_file
@@ -36,7 +36,7 @@
 
 # - name: permissions - Static
 #   file:
-#     path: '{{ host_mantisbt_docker_restore_dir }}{{ mantisbt_static_backup_file }}'
+#     path: '{{ mantisbt_docker_restore_dir }}{{ mantisbt_static_backup_file }}'
 #     state: touch
 #     owner: root
 #     group: root
@@ -44,7 +44,7 @@
 
 # - name: permissions - SQL (files table)
 #   file:
-#     path: '{{ host_mantisbt_docker_restore_dir }}{{ mantisbt_database_files_table_backup_file }}'
+#     path: '{{ mantisbt_docker_restore_dir }}{{ mantisbt_database_files_table_backup_file }}'
 #     state: touch
 #     owner: root
 #     group: root
@@ -52,7 +52,7 @@
 
 # - name: permissions - SQL (remaining tables)
 #   file:
-#     path: '{{ host_mantisbt_docker_restore_dir }}{{ mantisbt_database_backup_file }}'
+#     path: '{{ mantisbt_docker_restore_dir }}{{ mantisbt_database_backup_file }}'
 #     state: touch
 #     owner: root
 #     group: root
@@ -60,21 +60,21 @@
 
 - name: exists - Static
   stat:
-    path: '{{ host_mantisbt_docker_restore_dir }}{{ mantisbt_static_backup_file }}'
+    path: '{{ mantisbt_docker_restore_dir }}{{ mantisbt_static_backup_file }}'
     get_checksum: False
     get_md5: False
   register: st_mantisbt_static_archive
 
 - name: exists - SQL (files table)
   stat:
-    path: '{{ host_mantisbt_docker_restore_dir }}{{ mantisbt_database_files_table_backup_file }}'
+    path: '{{ mantisbt_docker_restore_dir }}{{ mantisbt_database_files_table_backup_file }}'
     get_checksum: False
     get_md5: False
   register: st_mantisbt_files_table_SQL_file
 
 - name: exists - SQL (remaining tables)
   stat:
-    path: '{{ host_mantisbt_docker_restore_dir }}{{ mantisbt_database_backup_file }}'
+    path: '{{ mantisbt_docker_restore_dir }}{{ mantisbt_database_backup_file }}'
     get_checksum: False
     get_md5: False
   register: st_mantisbt_SQL_file
@@ -93,7 +93,7 @@
 
 - name: Remove - Static and SQL
   file:
-    path: '{{ host_mantisbt_docker_restore_dir }}'
+    path: '{{ mantisbt_docker_restore_dir }}'
     state: absent
 
 - name: State file
diff --git a/roles/docker-mediawiki/defaults/main.yml b/roles/docker-mediawiki/defaults/main.yml
index dbfe971..7b88908 100644
--- a/roles/docker-mediawiki/defaults/main.yml
+++ b/roles/docker-mediawiki/defaults/main.yml
@@ -40,7 +40,7 @@ wiki_db_dv_name: '{{ docker_name_prefix }}wiki_db_DV'
 
 # restore directories to temporarly store data being restored into docker containers
 wiki_docker_backup_dir: '{{ docker_backup_dir }}/WIKI'
-host_wiki_docker_restore_dir: '{{ bacula_dest }}{{ wiki_docker_backup_dir }}'
+wiki_docker_restore_dir: '{{ bacula_dest }}{{ wiki_docker_backup_dir }}'
 
 # files restored from tape
 wiki_static_backup_file: '/mediawiki.tar'
diff --git a/roles/docker-mediawiki/tasks/main.yml b/roles/docker-mediawiki/tasks/main.yml
index 582a1fa..e16bb78 100644
--- a/roles/docker-mediawiki/tasks/main.yml
+++ b/roles/docker-mediawiki/tasks/main.yml
@@ -12,7 +12,7 @@
 - name: restore dir
   file:
     state: directory
-    path: '{{ host_wiki_docker_restore_dir }}'
+    path: '{{ wiki_docker_restore_dir }}'
     owner: root
     group: tape
     mode: 'u=rwx,g=rwx,o=rx'
diff --git a/roles/docker-mediawiki/tasks/restore.yml b/roles/docker-mediawiki/tasks/restore.yml
index 90b9b94..f2b3409 100644
--- a/roles/docker-mediawiki/tasks/restore.yml
+++ b/roles/docker-mediawiki/tasks/restore.yml
@@ -6,14 +6,14 @@
 
 - name: exists - Static
   stat:
-    path: '{{ host_wiki_docker_restore_dir }}{{ wiki_static_backup_file }}'
+    path: '{{ wiki_docker_restore_dir }}{{ wiki_static_backup_file }}'
     get_checksum: False
     get_md5: False
   register: st_wiki_static_archive
 
 - name: exists - SQL
   stat:
-    path: '{{ host_wiki_docker_restore_dir }}{{ wiki_database_backup_file }}'
+    path: '{{ wiki_docker_restore_dir }}{{ wiki_database_backup_file }}'
     get_checksum: False
     get_md5: False
   register: st_wiki_SQL_file
@@ -29,7 +29,7 @@
 
 # - name: permissions - Static
 #   file:
-#     path: '{{ host_wiki_docker_restore_dir }}{{ wiki_static_backup_file }}'
+#     path: '{{ wiki_docker_restore_dir }}{{ wiki_static_backup_file }}'
 #     state: touch
 #     owner: root
 #     group: root
@@ -37,7 +37,7 @@
 
 # - name: permissions - SQL
 #   file:
-#     path: '{{ host_wiki_docker_restore_dir }}{{ wiki_database_backup_file }}'
+#     path: '{{ wiki_docker_restore_dir }}{{ wiki_database_backup_file }}'
 #     state: touch
 #     owner: root
 #     group: root
@@ -45,14 +45,14 @@
 
 - name: exists - Static
   stat:
-    path: '{{ host_wiki_docker_restore_dir }}{{ wiki_static_backup_file }}'
+    path: '{{ wiki_docker_restore_dir }}{{ wiki_static_backup_file }}'
     get_checksum: False
     get_md5: False
   register: st_wiki_static_archive
 
 - name: exists - SQL
   stat:
-    path: '{{ host_wiki_docker_restore_dir }}{{ wiki_database_backup_file }}'
+    path: '{{ wiki_docker_restore_dir }}{{ wiki_database_backup_file }}'
     get_checksum: False
     get_md5: False
   register: st_wiki_SQL_file
@@ -71,7 +71,7 @@
 
 - name: Remove - Static and SQL
   file:
-    path: '{{ host_wiki_docker_restore_dir }}'
+    path: '{{ wiki_docker_restore_dir }}'
     state: absent
 
 - name: State file
diff --git a/roles/docker-openldap/defaults/main.yml b/roles/docker-openldap/defaults/main.yml
index 900100b..17b5044 100644
--- a/roles/docker-openldap/defaults/main.yml
+++ b/roles/docker-openldap/defaults/main.yml
@@ -34,14 +34,12 @@ openldap_image_name: '{{ docker_registry_username }}/ldap'
 openldap_container_name: '{{ docker_name_prefix }}ldap'
 
 # the name of the data-volume used by the container
-openldap_cv_name: '{{ docker_name_prefix }}ldap_CV'
-openldap_dv_name: '{{ docker_name_prefix }}ldap_DV'
+openldap_cv_name: '{{ docker_name_prefix }}openldap_CV'
+openldap_dv_name: '{{ docker_name_prefix }}openldap_DV'
 
 # restore directories to temporarly store data being restored into docker containers
 openldap_docker_backup_dir: '{{ docker_backup_dir }}/LDAP'
-host_openldap_docker_restore_dir: '{{ bacula_dest }}{{ openldap_docker_backup_dir }}'
-
-
+openldap_docker_restore_dir: '{{ bacula_dest }}{{ openldap_docker_backup_dir }}'
 
 # *****************************************************************************
 # Utility container - phpldapadmin
diff --git a/roles/docker-openldap/tasks/main.yml b/roles/docker-openldap/tasks/main.yml
index 45c83c2..9a2b5a1 100644
--- a/roles/docker-openldap/tasks/main.yml
+++ b/roles/docker-openldap/tasks/main.yml
@@ -5,8 +5,8 @@
 # A manuel step of setting the passwords into the LDAP database is required
 - assert:
     that:
-    - ldap_admin_password is defined
-    - ldap_proxyagent_password is defined
+    - openldap_admin_password is defined
+    - openldap_proxyagent_password is defined
 
 # *****************************************************************************
 # Setup the directory where the backup and restore is to take place
@@ -14,7 +14,7 @@
 - name: restore dir
   file:
     state: directory
-    path: '{{ host_openldap_docker_restore_dir }}'
+    path: '{{ openldap_docker_restore_dir }}'
     owner: root
     group: tape
     mode: 'u=rwx,g=rwx,o=rx'
@@ -174,7 +174,7 @@
     state: started
 
 - name: set LDAP admin password
-  command: 'docker exec -i {{ openldap_container_name }} /docker-entrypoint.sh set_admin_pass {{ ldap_admin_password | quote }}'
+  command: 'docker exec -i {{ openldap_container_name }} /docker-entrypoint.sh set_admin_pass {{ openldap_admin_password | quote }}'
 
 - name: set LDAP proxyagent password
-  command: 'docker exec -i {{ openldap_container_name }} /docker-entrypoint.sh set_proxyagent_pass {{ ldap_admin_password | quote }} {{ ldap_proxyagent_password | quote }}'
+  command: 'docker exec -i {{ openldap_container_name }} /docker-entrypoint.sh set_proxyagent_pass {{ openldap_admin_password | quote }} {{ openldap_proxyagent_password | quote }}'
diff --git a/roles/docker-openldap/tasks/restore.yml b/roles/docker-openldap/tasks/restore.yml
index 34c83e2..fd82c7f 100644
--- a/roles/docker-openldap/tasks/restore.yml
+++ b/roles/docker-openldap/tasks/restore.yml
@@ -6,8 +6,8 @@
 
 - name: find files
   files_in_dir:
-    path: '{{ host_openldap_docker_restore_dir }}'
-  register: ldap_backup
+    path: '{{ openldap_docker_restore_dir }}'
+  register: openldap_backup
 
 - name: Get data-volume data from tape
   bacula:
@@ -16,11 +16,11 @@
     fileset: '{{ bacula_fileset }}'
     dest: '{{ bacula_dest }}'
     path_to_restore: '{{ openldap_docker_backup_dir }}'
-  when: ldap_backup.file_list == []
+  when: openldap_backup.file_list == []
 
 # - name: Set permissions for data-volume data
 #   file:
-#     path: '{{ host_openldap_docker_restore_dir }}/{{ item.ldif_file }}'
+#     path: '{{ openldap_docker_restore_dir }}/{{ item.ldif_file }}'
 #     state: touch
 #     owner: root
 #     group: root
@@ -45,7 +45,7 @@
 
 - name: Remove database backup file
   file:
-    path: '{{ host_openldap_docker_restore_dir }}'
+    path: '{{ openldap_docker_restore_dir }}'
     state: absent
 
 - name: State file
diff --git a/roles/docker-openssl/defaults/main.yml b/roles/docker-openssl/defaults/main.yml
index 7e661a4..770403a 100644
--- a/roles/docker-openssl/defaults/main.yml
+++ b/roles/docker-openssl/defaults/main.yml
@@ -9,7 +9,7 @@ openssl_dv_name: '{{ docker_name_prefix }}openssl_DV'
 
 # restore directories to temporarly store data being restored into docker containers
 openssl_docker_backup_dir: '{{ docker_backup_dir }}/openssl'
-host_openssl_docker_restore_dir: '{{ bacula_dest }}{{ openssl_docker_backup_dir }}'
+openssl_docker_restore_dir: '{{ bacula_dest }}{{ openssl_docker_backup_dir }}'
 
 # files restored from tape
 openssl_backup_file: '/openssl.tar'
diff --git a/roles/docker-openssl/tasks/main.yml b/roles/docker-openssl/tasks/main.yml
index a3828f6..8f0eaf8 100644
--- a/roles/docker-openssl/tasks/main.yml
+++ b/roles/docker-openssl/tasks/main.yml
@@ -7,7 +7,7 @@
 - name: restore dir
   file:
     state: directory
-    path: '{{ host_openssl_docker_restore_dir }}'
+    path: '{{ openssl_docker_restore_dir }}'
     owner: root
     group: tape
     mode: 'u=rwx,g=rwx,o=rx'
diff --git a/roles/docker-openssl/tasks/restore.yml b/roles/docker-openssl/tasks/restore.yml
index 7740ffa..1cfc3d4 100644
--- a/roles/docker-openssl/tasks/restore.yml
+++ b/roles/docker-openssl/tasks/restore.yml
@@ -6,7 +6,7 @@
 
 - name: exists - openssl archive
   stat:
-    path: '{{ host_openssl_docker_restore_dir }}{{ openssl_backup_file }}'
+    path: '{{ openssl_docker_restore_dir }}{{ openssl_backup_file }}'
     get_checksum: False
     get_md5: False
   register: st_openssl_backup_file
@@ -22,7 +22,7 @@
 
 # - name: Set permissions for data-volume data
 #   file:
-#     path: '{{ host_openssl_docker_restore_dir }}{{ openssl_backup_file }}'
+#     path: '{{ openssl_docker_restore_dir }}{{ openssl_backup_file }}'
 #     state: touch
 #     owner: root
 #     group: root
@@ -30,7 +30,7 @@
 
 - name: exists - openssl archive
   stat:
-    path: '{{ host_openssl_docker_restore_dir }}{{ openssl_backup_file }}'
+    path: '{{ openssl_docker_restore_dir }}{{ openssl_backup_file }}'
     get_checksum: False
     get_md5: False
   register: st_openssl_backup_file
@@ -44,9 +44,9 @@
     image: '{{ openssl_image_name }}:{{ docker_image_tag }}'
     name: openssl_restore
     volumes:
-      - '{{ host_openssl_docker_restore_dir }}{{ openssl_backup_file }}:{{ host_openssl_docker_restore_dir }}{{ openssl_backup_file }}:z'
+      - '{{ openssl_docker_restore_dir }}{{ openssl_backup_file }}:{{ openssl_docker_restore_dir }}{{ openssl_backup_file }}:z'
       - '{{ openssl_dv_name }}:/etc/ssl/private:z'
-    command: ['extract', '{{ host_openssl_docker_restore_dir }}{{ openssl_backup_file }}']
+    command: ['extract', '{{ openssl_docker_restore_dir }}{{ openssl_backup_file }}']
 
 - name: extract archive
   docker_container:
@@ -59,7 +59,7 @@
 
 - name: remove - openssl archive
   file:
-    path: '{{ host_openssl_docker_restore_dir }}'
+    path: '{{ openssl_docker_restore_dir }}'
     state: absent
 
 - name: State file
diff --git a/roles/docker-svn/defaults/main.yml b/roles/docker-svn/defaults/main.yml
index bb41634..a18ab50 100644
--- a/roles/docker-svn/defaults/main.yml
+++ b/roles/docker-svn/defaults/main.yml
@@ -28,7 +28,7 @@ svn_dv_name: '{{ docker_name_prefix }}svn_DV'
 
 # restore directories to temporarly store data being restored into docker containers
 svn_docker_backup_dir: '{{ docker_backup_dir }}/SVN'
-host_svn_docker_restore_dir: '{{ bacula_dest }}{{ svn_docker_backup_dir }}'
+svn_docker_restore_dir: '{{ bacula_dest }}{{ svn_docker_backup_dir }}'
 
 # Network names
 docker_network_frontend: 'frontend'
diff --git a/roles/docker-svn/tasks/main.yml b/roles/docker-svn/tasks/main.yml
index 40661b6..e39e3f9 100644
--- a/roles/docker-svn/tasks/main.yml
+++ b/roles/docker-svn/tasks/main.yml
@@ -7,7 +7,7 @@
 - name: restore dir
   file:
     state: directory
-    path: '{{ host_svn_docker_restore_dir }}'
+    path: '{{ svn_docker_restore_dir }}'
     owner: root
     group: tape
     mode: 'u=rwx,g=rwx,o=rx'
@@ -90,7 +90,7 @@
       - '{{ svn_dv_name }}:/var/lib/svn:z'
     env:
       SVN_HOSTNAME: '{{ container_addr_map.svn.hostname }}'
-      SVN_LDAP_PASSWORD: '{{ ldap_proxyagent_password }}'
+      SVN_LDAP_PASSWORD: '{{ openldap_proxyagent_password }}'
     image: '{{ svn_image_name }}:{{ docker_image_tag }}'
 
 # *****************************************************************************
diff --git a/roles/docker-svn/tasks/restore.yml b/roles/docker-svn/tasks/restore.yml
index f8bf2a2..22e7204 100644
--- a/roles/docker-svn/tasks/restore.yml
+++ b/roles/docker-svn/tasks/restore.yml
@@ -6,7 +6,7 @@
 
 - name: find files
   files_in_dir:
-    path: '{{ host_svn_docker_restore_dir }}'
+    path: '{{ svn_docker_restore_dir }}'
   register: svn_repo_backup
 
 - name: data from tape
@@ -20,12 +20,12 @@
 
 - name: find files
   files_in_dir:
-    path: '{{ host_svn_docker_restore_dir }}'
+    path: '{{ svn_docker_restore_dir }}'
   register: svn_repo_backup
 
 # - name: File permissions
 #   file:
-#     path: '{{ host_svn_docker_restore_dir }}/{{ item }}'
+#     path: '{{ svn_docker_restore_dir }}/{{ item }}'
 #     state: touch
 #     owner: root
 #     group: root
@@ -40,7 +40,7 @@
     detach: False
     volumes:
       - '{{ svn_dv_name }}:/var/lib/svn:z'
-      - '{{ host_svn_docker_restore_dir }}:/tmp/import_export:z'
+      - '{{ svn_docker_restore_dir }}:/tmp/import_export:z'
     image: '{{ svn_image_name }}:{{ docker_image_tag }}'
     command: ['import', '{{ svn_repos }}']
     name: 'svn_restore'
@@ -55,7 +55,7 @@
 
 - name: Remove files
   file:
-    path: '{{ host_svn_docker_restore_dir }}'
+    path: '{{ svn_docker_restore_dir }}'
     state: absent
 
 - name: State file
